sagemaker-triton/ensemble/tf-dali-ensemble-cv.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>This GitHub sample image link does not work since you haven't added the image to your PR (which is good, we avoid committing data directly to the repo). I recommend you use the S3 bucket link instead: <a href="https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/pets/shiba_inu_dog.jpg" target="_blank">https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/pets/shiba_inu_dog.jpg</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
fixed the url